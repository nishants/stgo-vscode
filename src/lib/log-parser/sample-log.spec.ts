import parser from './index';
import * as sampleData from './sample-log.json';

describe('lib', () => {
  test('if utils mocked automatically', () => {
    expect(parser.parse(sampleData)).toMatchSnapshot();
  });
});