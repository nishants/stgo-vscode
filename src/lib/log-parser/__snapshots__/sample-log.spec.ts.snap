// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lib if utils mocked automatically 1`] = `
Object {
  "addCommit": [Function],
  "getCommits": [Function],
}
`;
